docker-compose.yml

```diff
@@ -90,9 +90,9 @@ services:
       - postgres
     environment:
       - DB_HOST=postgres
-      - DB_USER=postgres
-      - DB_PASSWORD=postgres
-      - DB_DATABASE=metadata_db
+      - DB_USER=metdata_user
+      - DB_PASSWORD=metdata_pass
+      - DB_DATABASE=metadata
```

Comment:
```suggestion
      - DB_USER=metadata_user
      - DB_PASSWORD=metadata_pass
      - DB_DATABASE=metadata
```

Reply:
Thank you for catching these 🙃 ✏️ (and curse my typing abilities!)